assets/app/view/game/hex.rb

```diff
@@ -141,6 +141,15 @@ def on_hex_click
         case @role
         when :map
           return process_action(Engine::Action::Assign.new(@entity, target: @hex)) if @actions.include?('assign')
+
+          # passing the entity to active_step yields EndGame as the active step for some reason, so we pass nil
```

Comment:
uhhh what? sounds like a hack, can you figure this out and do it properly?

Reply:
I did some poking around and it looks like the "normal" way of invoking `active_step` is to pass nothing (which defaults `entity` to `nil` anyways) and the times that it is given a parameter it is given a `Company` so the `EndGame` actions are empty

```
      def actions(entity)
        return [] if entity.company?
```

Reply:
ok can you remove the comment and just call it the normal way? round.active_step

Reply:
Oops! The push I made in the morning with the change failed on invalid password, it should be fixed now